8.8/vips.modules

```diff
@@ -657,34 +568,22 @@
 
   <!-- libMagick++.dll fails to link due to some exception problem, disable it
 
-       disable largefile support, we rely on vips for that and ImageMagick's 
+       disable largefile support, we rely on vips for that and ImageMagick's
        detection does not work when cross-compiling
 
        disable openmp, ubuntu 14.04 mingw has a broken libgomp
 
-       6.9.0-10 and later don't build under mingw
-
-       6.9.2-3 confirmed to also fail, some sort of locale issue
-
-       6.9.3-10 and later use intsafe.h which does not yet have an 
-       implementation on mingw
-
        7.x is much slower (though libvips does support it)
 
-       since we are forced to use a very old version, don't use vips + IM on 
-       Windows if you are at all concerned about security :( 
-
     -->
 
-  <autotools id="imagemagick" 
+  <autotools id="imagemagick"
     autogen-sh="configure"
-    autogenargs="--without-threads --without-x --with-modules=no --with-openexr=no --without-gvc --without-lqr --without-magick-plus-plus --disable-largefile --without-rsvg --disable-openmp"
-    >
+    autogenargs="--without-x --without-lzma --without-modules --without-openexr --without-heic --without-gvc --without-lqr --without-magick-plus-plus --disable-largefile --without-rsvg --without-zlib --disable-openmp">
     <branch
       repo="magick"
-      module="ImageMagick-6.8.9-10.tar.xz"
-      version="6.8.9"
-    />
+      module="ImageMagick-6.9.10-42.tar.xz"
+      version="6.9.10-42"/>
```

Comment:
Version bumped a 4-year-old dependency, it'll reduce the attack surface (the latest versions of ImageMagick are continuous fuzzed by [OSS-Fuzz](https://github.com/google/oss-fuzz)).

The only disadvantage of this version bump is that it can suddenly break a build, see:
https://github.com/libvips/build-win64-mxe/issues/7

(ImageMagick removes older versions on their website).

Reply:
We were stuck on an old imagemagick because of intsafe. I guess this in mingw now?

Reply:
I think this was fixed with ImageMagick 6.9.5-3 (released on 22 July 2016). They have excluded the `intsafe.h` header for MinGW-w64:
https://github.com/ImageMagick/ImageMagick6/commit/23a66459502c61811db2e1f4d6773916bea87b3e#diff-6a582070a57b62127f59170511cae756R25